CMake/curl-config.cmake.in

```diff
@@ -1,9 +1,15 @@
 @PACKAGE_INIT@
 
-if("@USE_OPENSSL@")
+if("@USE_OPENSSL@" OR "@CURL_ZLIB@" )
     include(CMakeFindDependencyMacro)
-    find_dependency(OpenSSL "@OPENSSL_VERSION_MAJOR@")
+    if ("@USE_OPENSSL@")
+        find_dependency(OpenSSL "@OPENSSL_VERSION_MAJOR@")
+    endif()
```

Comment:
AFAIR, indentation used in curl project is 2 spaces. Could you please fix? Looks like it has lurked from the previous review.